docs/FAQ.rst

```diff
@@ -143,6 +143,24 @@ How to I resolve a build error for Avalon?
   or the
   `chat channel <https://chat.hyperledger.org/channel/avalon>`_
 
+How do I fix this error installing Avalon on Azure ACC with Intel SGX: "/dev/isgx": no such file or directory
+--------------------------------------------------------------------------------------------------------------------------------------
+Azure Confidential Computing (ACC) installs the ``/dev/sgx`` driver
+on their cloud instances with Intel SGX.
+Avalon uses ``/dev/isgx`` instead.
+So if you use Avalon with Intel SGX hardware mode enabled
+on ACC without ``/dev/isgx`` installed you get this error:
```

Comment:
This is correct. The commit message needs to be modified --> MS Azure Confidential Computing (ACC) installs /dev/isgx, but Avalon needs /dev/sgx. Change this to MS Azure Confidential Computing (ACC) installs /dev/sgx, but Avalon needs /dev/isgx

Reply:
@manojgop Fixed /dev/sgx <--> /dev/isgx in commit message.